platform/x11/godot_x11.cpp

```diff
@@ -26,13 +26,17 @@
 /* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
+#include <unistd.h>
 #include "main/main.h"
 #include "os_x11.h"
 
 int main(int argc, char* argv[]) {
 
 	OS_X11 os;
 
+	char *cwd = (char*)malloc(PATH_MAX);
+    getcwd(cwd, PATH_MAX);
```

Comment:
But I'd still have to set the size to PATH_MAX before and just use it as a char*. I don't think it'll be any prettier